met-web/src/apiManager/endpoints/index.ts

```diff
@@ -0,0 +1,9 @@
+import { AppConfig } from '../../config';
+
+const API = {
+    MET_GET_ENGAGEMENTS: `${AppConfig.apiUrl}/engagement/`,
+    MET_GET_ENGAGEMENT: `${AppConfig.apiUrl}/engagement/<engagement_id>`,
+    MET_CREATE_ENGAGEMENT: `${AppConfig.apiUrl}/engagement/`,
+    MET_UPDATE_ENGAGEMENT: `${AppConfig.apiUrl}/engagement/`,
+};
```

Comment:
Thanks for doing this..Much needed change..
But I think its safe to go with one URL ..99% we will have one rest end point at backend... so MET_ENGAGEMENTS will be good enough to support all rest verbs on that url..
Also do we need MET_? MET is kind of implicit..

Reply:
I have a similar feeling but just feel it's difficult to name an endpoint and use for single and multiple operations. Like MET_ENGAGEMENTS or ENGAGEMENTS and use it for getting all engagements but also for creating one single engagement. What do you think?